pkg/endpoint/endpoint.go

```diff
@@ -2637,8 +2637,10 @@ func (e *Endpoint) syncPolicyMapController() {
 	e.controllers.UpdateController(ctrlName,
 		controller.ControllerParams{
 			DoFunc: func() (reterr error) {
+				// Failure to lock is not an error, it means
```

Comment:
The only nit that I have here is that this isn't CNP sync, it's PolicyMap sync - can you update the commit title to reflect this?